gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -112,6 +112,10 @@ CreateConVar("ttt_credits_award_repeat", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
 -- @realm server
 CreateConVar("ttt_credits_award_kill", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
 
+---
+-- @realm server
+local session_limits_enabled = CreateConVar("ttt_session_limits_enabled", "1", SERVER and {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED} or FCVAR_REPLICATED)
```

Comment:
```suggestion
local ttt_session_limits_enabled = CreateConVar("ttt_session_limits_enabled", "1", SERVER and {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED} or FCVAR_REPLICATED)
```

Reply:
All the other local variables leave out the prefix as well.

Reply:
Oh, you're right they are inconsistent. 
But nonetheless i think it is better this way, since it now is the same as the convar.
Thanks!